### ISSUE TITLE
Source code for packages

### ISSUE THREAD

**Issue by Scorpion2185** [vector-im/element-web#21300]
#### What would you like to do?
To have the source code of the packages at https://packages.element.io/debian/. 
Debian repos also have the source of the packages.
#### Why would you like to do it?
To add some option when building it with `dpkg-buildpackage`.

**Comment by dbkr** [comment]
This could really do with some further explanation. Where are you envisaging the source would be that it isn't currently?

**Comment by Scorpion2185** [comment]
I mean the source of the **package** already built for the distro.

If I understood correctly the line `deb https://packages.element.io/debian/ default main` is added to the apt sources.
There is no `deb-src https://packages.element.io/debian/ default main` entry that provides the source of the packages.

I didn't add the Element repo yet.

**Comment by Scorpion2185** [comment]
@dbkr is that correct?

**Comment by dbkr** [comment]
OK, so this is asking for a deb-src repository with the source code for each version, (so debian users could get the source using `apt-get source element-desktop`), if I've understood correctly. This would also be slightly complex because the desktop packaging process fetches a prebuilt copy of element-web, so actually getting all the source in a form that could be built into a debian package would not be trivial.

**Comment by dbkr** [comment]
Also, this issue could really do with some actual use-cases, ie. what would having this allow people to do? Element isn't a straightforward tarball of c source code, so it's not really the case that having source packages would allow it to be built for different architectures.

**Comment by Scorpion2185** [comment]
Yes it is for a `deb-src` repository with the source code for each version, (so debian users can get the source using `apt-get source element-desktop`).
And the same for any other package version I think that could be useful.

Some people always prefer to compile instead of installing.

In my case I want to try to take a look at this: https://github.com/vector-im/element-web/issues/19792.
Simply adding `element-desktop --no-sandbox` doesn't work.
